### ISSUE TITLE
No more keyboard shortcuts for quick search

### ISSUE THREAD

**Issue by tsez** [FreeRDP/Remmina#887]
Hello,
Ctrl+F and TAB keyboard shortcuts were disabled in the new version. I cannot search and connect with keyboard only. May be related to #875.

- Ctrl+F: give focus to search bar.
- TAB: give focus to filtered connections list.
- Enter: connect to selected connection.

Xubuntu 16.04
Remmina 1.2.0-rcgit.13

**Comment by antenore** [comment]
It's related, yes.

We completely changed the main window interface to make it a bit cleaner (I hope you like).

I've tried in a first instance to keep the shortcuts but it wasn't working.

I'll take a look asap.

BTW, when you open the remmina main window, the focus should be already in the search bar and if you hit TAB it should move the tree list. If it's not the case there is a problem (it works on my machine for instance).

**Comment by tsez** [comment]
Yes I like the merge of Quick Connect and Search, it is a good idea :)

When I start Remmina I don't have the focus on Search but its ok for me. I just need a shortcut to give the focus on Search. 
I give the focus on the list when I press TABx2.

Thank you.

**Comment by antenore** [comment]
Would you mind testing the branch remmina_main_fixes?

Otherwise, to make it short, now, the first time you open Remmina, the cursor is inside the search entry bar, waiting for your input.
When you hit ctrl+f inside the remmina main window, the cursor is placed again inside the search bar.
To pass from the search bar to the tree view, you still have to hit TABx2

To change the Tab behaviour is quite complicated, but consider that in the future I'd like to change even further the Remmina Main.

**Comment by antenore** [comment]
@giox069 if you have time to test as well...

**Comment by Chugajstyr** [comment]
Hi! When I open Remmina main window, focus doesn`t in search bar, only after double TAB focus moved to. Ctrl+F still doesn`t work. Linux Mint 17.3 (MATE Desktop 1.12), Remmina 1.2.0-rcgit.13

**Comment by antenore** [comment]
I think your remmina it's not really updated.

Find a the file remmina_main.glade ( i.e. whit ```locate remmina_main.glade``` ) and open it with an editor.

Can you check if you have this code inside it?

```xml
    282                  <child>
   283                    <object class="GtkEntry" id="entry_quick_connect_server">
   284                      <property name="visible">True</property>
   285                      <property name="can_focus">True</property>
   286                      <property name="has_focus">True</property>
   287                      <property name="is_focus">True</property>
   288                      <property name="can_default">True</property>
   289                      <property name="has_default">True</property>
   290                      <property name="receives_default">True</property>
   291                      <property name="tooltip_text" translatable="yes">search string or server name/ip address for quick connect</property>
   292                      <property name="hexpand">True</property>
   293                      <property name="width_chars">25</property>
   294                      <property name="primary_icon_stock">gtk-connect</property>
   295                      <property name="secondary_icon_stock">gtk-clear</property>
   296                      <property name="completion">entrycompletion</property>
   297                      <property name="input_hints">GTK_INPUT_HINT_WORD_COMPLETION | GTK_INPUT_HINT_NONE</property>
   298                      <signal name="activate" handler="remmina_main_quickconnect_on_click" swapped="no"/>
   299                      <signal name="changed" handler="remmina_main_quick_search_on_changed" swapped="no"/>
   300                      <signal name="icon-press" handler="remmina_main_quick_search_on_icon_press" swapped="no"/>
   301                      <signal name="insert-at-cursor" handler="remmina_main_load_files_cb" swapped="no"/>
   302                      <accelerator key="f" signal="grab-focus" modifiers="GDK_CONTROL_MASK"/>
   303                    </object>
   304                    <packing>
   305                      <property name="expand">True</property>
   306                      <property name="fill">True</property>
   307                      <property name="position">2</property>
   308                    </packing>
   309                  </child>
```

If you don't have all the **focus** properties it means you have still a previous version.

**Comment by tsez** [comment]
Hello, I just upgraded Remmina and did a test:
- default focus to quick search: ok
- ctrl+f shortcut: ok

All is fine for me, thank you.

**Comment by antenore** [comment]
Thanks @tsez

**Comment by Chugajstyr** [comment]
I can confirm - after Remmina update default focus in search bar and Ctrl+F shortcut works.
